administration/src/util/translations/de.json

```diff
@@ -41,6 +41,7 @@
     "responsibility": "Tätigkeit",
     "amountOfWork": "Arbeitsstunden pro Woche (Durchschnitt)",
     "workSinceDate": "Tätig seit",
+    "isAlreadyVerified": "Die Angaben wurden bereits durch den zuständigen Verein bei Beantragung verifiziert",
```

Comment:
❓ i couldn't find the place where this translation is used
Maybe in the `VerificationsView` below the headline? Not sure if it fits in the tooltip in the `PreverifiedQuickIndicator`

Reply:
it is in the application detail view in digitale druckerei -> eingegangende Anträge -> expand details of the application

Reply:
okay sorry not sure why i didn't find this via search. thx for the hint